recipes/metaknowledge/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "metaknowledge" %}
+{% set version = "3.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f137dce7ed7e1fb3a73730716d37e2b5757d9fc94f3bb50f8e82338c38b17dae
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - metaknowledge = metaknowledge.bin:mkCLI
```

From the `setup.py` it looks like you need three entry points?
```suggestion
    - metaknowledge = metaknowledge.bin:mkCLI
    - metaknowledge-mdToNb = metaknowledge.bin:mkMdToNb
    - metaknowledge-DocsGen = metaknowledge.bin:mkDocs
```

---

Yes, I left them out because they weren't needed to prevent the `entry_point` error. I can put it in of course, but I wasn't sure whether that would be needed. If it is needed for the proper functioning of it, I'd be glad to include it of course.

---

I'm not sure if they're strictly needed and I don't really understand what does and doesn't trigger that Windows error but I would add them to be safe.

---

OK, will do, thanks.